playground/app/app.config.ts

```diff
@@ -1,9 +1,17 @@
 export default defineAppConfig({
   toaster: {
-    position: 'bottom-right' as const,
+    position: 'bottom-right',
```

Comment:
This should be kept:
```suggestion
    position: 'bottom-right' as const,
```

Reply:
ah yes, unwanted change.